server/athenian/api/controllers/goal_controller.py

```diff
@@ -429,3 +436,40 @@ async def _parse_update_request(
         jira_priorities=jira_priorities,
         jira_issue_types=jira_issue_types,
     )
+
+
```

Comment:
IMO this looks nicer unless we'll have to update this frequently. Probably, we'll move them somewhere up where the metrics live, even.